packages/graphql/src/schema/resolvers/create.ts

```diff
@@ -29,7 +29,7 @@ export default function createResolver({ node }: { node: Node }) {
     async function resolve(_root: any, _args: any, _context: unknown, info: GraphQLResolveInfo) {
```

Comment:
I think the underscore could be removed here, which means you can just do `translateCreate({ context, node, args });` below. It was just this way because it was unused.

```suggestion
    async function resolve(_root: any, args: any, _context: unknown, info: GraphQLResolveInfo) {
```

Reply:
Oh nice!